services/online_exam_service/.env

```
@@ -1,18 +1,54 @@
+
+# BLOCKCHAIN CONFIG (Giữ nguyên các giá trị của bạn)
 WEB3_PROVIDER_URL=http://127.0.0.1:7545
 GRADE_LEDGER_CONTRACT_ADDRESS=0xFf72998a0e9178Aa2890B82F90bAB2De0Cd118b0
-OWNER_ACCOUNT_PRIVATE_KEY=0x80801b1f0a74e06a6bc872b62b08ad7c9b17d4477f7d7cf5fe313f99573e7a2d
+OWNER_ACCOUNT_PRIVATE_KEY=0x7dde22895f567fa99c16102977ac17470470873768b3351c7cce1c5a1541d39a
+BLOCKCHAIN_CHAIN_ID=1337
 
 # OTHER SERVICES URL
 PROCTORING_SERVICE_URL=http://127.0.0.1:8082
-JWT_SECRET=mySecretKey12345678901234567890123456789012345678901234567890
+IDENTITY_SERVICE_URL=http://localhost:8081
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The IDENTITY_SERVICE_URL key should go before the PROCTORING_SERVICE_URL key

(UnorderedKey)